src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -83,7 +83,7 @@ export class MdTabLink {
 })
 export class MdTabLinkRipple extends MdRipple {
   constructor(elementRef: ElementRef, ngZone: NgZone, ruler: ViewportRuler,
-              @Optional() @Inject(MD_DISABLE_RIPPLES) forceDisableRipples: boolean) {
+              @Optional() @Inject(MD_RIPPLE_GLOBAL_OPTIONS) forceDisableRipples: boolean) {
```

Comment:
I was just about to submit a PR for this :D. I think the parameter here has to be renamed as well.